app/scripts/modules/amazon/src/loadBalancer/details/LoadBalancerActions.tsx

```diff
@@ -121,15 +122,13 @@ export class LoadBalancerActions extends React.Component<ILoadBalancerActionsPro
           </Dropdown.Toggle>
           <Dropdown.Menu className="dropdown-menu">
             {application && (
-              <li>
-                <a className="clickable" onClick={this.editLoadBalancer}>
-                  Edit Load Balancer
-                </a>
-              </li>
+              <ManagedMenuItem resource={loadBalancer} application={app} onClick={this.editLoadBalancer}>
+                Edit Load Balancer
+              </ManagedMenuItem>
             )}
             {!application && (
               <li className="disabled">
-                <a className="clickable" onClick={this.editLoadBalancer}>
```

Comment:
UGH thanks for fixing this, we hear about it every few months and then willfully ignore it.